### DISCUSSION TITLE
idk how to open the app

### DISCUSSION THREAD

**Discussion by hellohihihiiihhello** [ruffle-rs/ruffle#828] [General]
idk how to open the app

**Comment by zanzlanz** [comment]
Check out the [Wiki](https://github.com/ruffle-rs/ruffle/wiki/Using-Ruffle), the [build instructions](https://github.com/ruffle-rs/ruffle/wiki/Building-Ruffle), or even the [live online demo](http://ruffle.rs/demo) to get started.

Otherwise, do you have any other details about what you're trying to do?

**Comment by Herschel** [comment]
[moved to discussions since this is a support request]

The easiest way to test ruffle is on the web demo page. You can upload your own SWF there to try it:
http://ruffle-rs.s3-website-us-west-1.amazonaws.com/builds/web-demo/

You can grab the latest desktop build of Ruffle here:
http://ruffle-rs.s3-website-us-west-1.amazonaws.com/windows/
Note that this is a command-line app, so you have to pass it an SWF file as a parameter:
ruffle my_file.test

**Reply by zanzlanz** [reply]
It is convenient to simply drag the SWF over Ruffle's exe as well.

**Comment by hellohihihiiihhello** [comment]
?????


Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10
